roms_tools/setup/river_forcing.py

```diff
@@ -674,7 +675,24 @@ def to_yaml(self, filepath: Union[str, Path]) -> None:
             The path to the YAML file where the parameters will be saved.
         """
 
-        _to_yaml(self, filepath)
+        # Serialize object into dictionary
+        yaml_data = _to_dict(self)
```

Comment:
`yaml_data` is not yaml. consider renaming to something like `forcing_dict` and there is no need to tell me the output of `_to_dict` is a dictionary with an extra comment.